templates/vue/src/components/TapestryMain/TapestryNode/index.vue

```diff
@@ -90,7 +90,9 @@
           >
             <tapestry-icon :icon="icon" svg></tapestry-icon>
           </node-button>
-          <template v-if="isLoggedIn">
+          <template
+            v-if="(isLoggedIn && this.settings.draftNodesEnabled) || canEditTapestry"
```

Comment:
This is not correct logic as it is going to prevent anyone who has been given add/edit access to a node not to be able to add/edit.